src/App.js

```diff
@@ -1,29 +1,26 @@
 import React from 'react';
-
+import { PostList } from './components/PostList';
 import './App.scss';
 
 import posts from './api/posts';
 import comments from './api/comments';
 import users from './api/users';
 
-const App = () => (
-  <div className="App">
-    <h1>Static list of posts</h1>
-
-    <p>
-      <span>posts: </span>
-      {posts.length}
-    </p>
-
-    <p>
-      <span>comments: </span>
-      {comments.length}
-    </p>
+const unitedPost = posts.map(
+  post => ({
+    ...post,
+    user: users.find(
+      user => user.id === post.userId,
+    ),
+    comment: comments.find(
+      comment => comment.postId === post.id,
+    ),
```

Comment:
```suggestion
    comment: comments.filter(
      comment => comment.postId === post.id,
    ),
```

Тобі потрібно знайти всі коменти